src/main/java/com/powsybl/openloadflow/network/impl/LfShuntImpl.java

```diff
@@ -18,17 +20,16 @@
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-public class LfShuntImpl implements LfShunt {
+public class LfShuntImpl extends AbstractElement implements LfShunt {
```

Comment:
For now there's no use for `LfShuntImpl` to extend `AbstractElement`, but I suppose you're anticipating some feature?
I'm asking as if we don't extend it, it would give more consistency to `AbstractElement` in a way (and some more `LfBus` / `LfBranch` common methods could be added)

Reply:
Honestly, I only added AbstractElement to fix a Sonar duplicated code issue at the moment. But for sure we can probably share more code. I Propose to do it in an other PR as it was really not the aim of this one.